python/instrumentation/openinference-instrumentation-llama-index/src/openinference/instrumentation/llama_index/_handler.py

```diff
@@ -910,6 +915,37 @@ def _asdict(obj: Any) -> Any:
             return repr(obj)
 
 
+def _get_attributes_from_message_content(
+    content: Mapping[str, Any],
+) -> Iterator[Tuple[str, AttributeValue]]:
+    content = dict(content)
+    type_ = content.pop("type")
```

Comment:
i'm surprised the type checker didn't pick this up, but `Mapping` is not supposed to be mutable.
```suggestion
    type_ = content.get("type")
```

Reply:
Oh good call